client/index.js

```diff
@@ -5,18 +5,15 @@ const MerkleMineGenerator = require("./lib/MerkleMineGenerator")
 const { makeTree } = require("./lib/helpers")
 
 const argv = require("yargs")
-      .usage("Usage: $0 --rinkeby --dev --acctFile [accounts file] --datadir [data directory] --merkleMine [MerkleMine address] --recipient [recipient address] --caller [caller address] --gasPrice [gas price]")
-      .boolean(["rinkeby", "dev"])
+      .usage("Usage: $0 --rinkeby --dev --generate --acctFile [accounts file] --datadir [data directory] --merkleMine [MerkleMine address] --recipient [recipient address] --caller [caller address] --gasPrice [gas price]")
+      .boolean(["rinkeby", "dev", "generate"])
+      .string(["merkleMine", "recipient", "caller"])
       .default("gasPrice", 5000000000)
```

Comment:
should we also set a default for the merkleMine address after we have it? I know it makes it less generic, but it might help usability a lot.